### PR TITLE
Added Styled Text to user page

### PR DESCRIPTION
**Pull request by zarihaidarian**
Added Styled Text 1 and some of Styled Text 2. Edited user-page.html, user-page-loader.js, and MessageServlet.java to edit messages using basic html.

### CODE REVIEW

**Comment by al-fred-o** [comment]
Based on your PR description, I cannot tell what the changes you made actually affect. Maybe add some screenshots of before and after?

File: src/main/webapp/js/user-page-loader.js [hunk 1]
```diff
@@ -92,6 +92,7 @@ function buildMessageDiv(message) {
 
 /** Fetches data and populates the UI of the page. */
 function buildUI() {
+  ClassicEditor.create( document.getElementById('message-input') );
```
Comment (al-fred-o) [inline src/main/webapp/js/user-page-loader.js:95]: Code seems fine syntactically, but it is unclear what the new code you added actually does compared to what was already there.

File: src/main/java/com/google/codeu/servlets/MessageServlet.java [hunk 2]
```diff
@@ -76,7 +76,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
 
     String user = userService.getCurrentUser().getEmail();
-    String userText = Jsoup.clean(request.getParameter("text"), Whitelist.none());
```
Comment (al-fred-o) [inline src/main/java/com/google/codeu/servlets/MessageServlet.java:79]: Again, I am not quite sure what this change does.

**Review by al-fred-o — APPROVED** [review]
I think you just need to comment a little bit more to make it clear what you Java code does and you should be good to go!

File: src/main/webapp/js/user-page-loader.js (hunk 1 above)
Comment (zarihaidarian) [reply src/main/webapp/js/user-page-loader.js:95]: Sorry for the lack of description. This line of javascript comes from the CKEditor. It's what changes the text editor from a plain text editor to a rich text editor. When running the website you should now be able to do basic html editing when writing something in the text box.

File: src/main/java/com/google/codeu/servlets/MessageServlet.java (hunk 2 above)
Comment (zarihaidarian) [reply src/main/java/com/google/codeu/servlets/MessageServlet.java:79]: This line of Java control what html the user can and cant use when writing messages. Using a Java library called Jsoup we can let users edit their text with html but only allowing certain basic tags to be used. I changed the line to Whitelist.basicWithImages to restrict the amount of html the user can use.

**Pull request merged**